src/services/crons/price.discovery.cache.warmer.service.ts

```diff
@@ -126,7 +127,8 @@ export class PriceDiscoveryCacheWarmerService {
         }
     }
 
-    @Cron('*/12 * * * * *') // Update prices and reserves every 6 seconds
+    @Cron(CronExpression.EVERY_MINUTE) // Update prices and reserves every 6 seconds
```

Comment:
comment is wrong, according to the cron expression

Reply:
done